Permission.podspec

```diff
@@ -64,6 +64,11 @@ Pod::Spec.new do |s|
     no.pod_target_xcconfig = { "SWIFT_ACTIVE_COMPILATION_CONDITIONS"  => "PERMISSION_NOTIFICATIONS" }
   end
 
+  s.subspec 'UserNotifications' do |no|
+    no.dependency 'Permission/Core'
+    no.pod_target_xcconfig = { "SWIFT_ACTIVE_COMPILATION_CONDITIONS"  => "PERMISSION_USERNOTIFICATIONS" }
```

Comment:
Could we rename `PERMISSION_USERNOTIFICATIONS` to `PERMISSION_USER_NOTIFICATIONS` for the sake of consistency please?